python_tutorial/README.md

```diff
@@ -0,0 +1,1017 @@
+
+### Windows basics
+Since you are on Windows, you will use the *Anaconda prompt* that is installed with Conda. In this terminal, the commands are different from those of UNIX.
+
+Download **[LINK TO ZIP]()** into a directory of your choice, unzip it using your file explorer and change the current directory of the *Anaconda prompt* to this new directory.
+
+You can find several commands to manipulate files and the current directory of your prompt in the following:
+
+- If you are not on the desired disk, you just have to type its name in the prompt:
+	
+```
+C:\Users>D:
+D:\>
+```
+
+- `md` to create a new directory
+- `cd` to change directory
+- `dir` to see a listing of the contents of a directory
+- `copy` to copy a file or files
+- `del` to remove (delete) a file
+- `move` to move a file
+- `echo %cd%` prints your current path
+- To create an empty file, prefer the file explorer
+
+### Python environment management
+In the following, the commands are the same either you are working on Anaconda prompt or a UNIX terminal.
+
+#### Creating a Conda Environment
+The command for creating a conda environment with Python 3.6 is:
+
+```
+conda create --name <env-name> python=3.6
+```
+
+For us, we decide to name our environment cs188, so we run the following command, and press y to confirm installing any missing packages.
+
+```
+MacBook-Pro-de-Pascal:python_basics pascalleroy$ conda create --name info8006 python=3.6
+```
+
+#### Entering the Environment
+To enter the conda environment that we just created, do the following. Note that the Python version within the environment is 3.6, just what we want.
+
+```
+MacBook-Pro-de-Pascal:python_basics pascalleroy$ conda activate info8006
+(info8006) MacBook-Pro-de-Pascal:python_basics pascalleroy$ python -V
+Python 3.6.9 :: Anaconda, Inc.
+```
+
+Note: the tag (<env-name>) shows you the name of the conda environment that is active. In our case, we have (info8006), as what we’d expect.
+
+#### Leaving the Environment
+Leaving the environment is just as easy.
+```
+(info8006) MacBook-Pro-de-Pascal:python_basics pascalleroy$ conda deactivate
+MacBook-Pro-de-Pascal:python_basics pascalleroy$ python -V
+Python 2.7.10
+```
+
+Our python version has now returned to whatever the system default is!
+
+## Python Basics
+### Required Files
+Following the terminal basics of your OS, you have normally dowloaded a zip file, extracted the content of this archive in a chosen directory and the current directoty of your terminal should contains those files. 
+
+### Introduction
+
+The programming assignments in this course will be written in Python, an interpreted, object-oriented language that shares some features with both Java and Scheme. This tutorial will walk through the primary syntactic constructions in Python, using short examples.
+
+We encourage you to type all python shown in the tutorial onto your own machine. Make sure it responds the same way.

```

Comment:
We encourage you to type in yourself the Python code of this tutorial. Make sure it produces the same results.